Signal/src/util/Pastelog.h

```diff
@@ -0,0 +1,16 @@
+//
```

Comment:
Signal/src/util/Pastelog...

Most projects I've worked on in the paste have isolated third party code in a separate directory (e.g. Signal/src/ThirdParty/Pastelog.m).  a) we can modify the precommit script to ignore this directory, to make it easier to take patches from the source. b) this makes the origin of the code clear.